lib/reek/cli/options.rb

```diff
@@ -9,7 +9,7 @@ module CLI
     #
     # See {file:docs/Command-Line-Options.md} for details.
     #
-    # :reek:TooManyInstanceVariables: { max_instance_variables: 9 }
+    # :reek:TooManyInstanceVariables: { max_instance_variables: 10 }
```

Comment:
how could this be solved? replacing the bunch of instance variables with a struct? class variables?